fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonUtils.xtend

```diff
@@ -292,7 +292,7 @@ class PythonUtils {
     def String checkEnumLabels (Attribute attr){
     	var flag = "valid"
     	for(label: attr.enumLabels){
-    	 	if(label.contains("-")||label.contains("!")||label.contains("#")||label.contains("@")||label.contains("%")||label.contains("$")||label.charAt(0).toString.matches("[0-9]")){
+    	 	if(label.contains("-")||label.contains("+")||label.contains("=")||label.contains("%")||label.contains("*")||label.charAt(0).toString.matches("[0-9]")){
```

Comment:
Why the change to the characters?  Agree that `+=*` are problems, but `!#@$` seem problematic too.   Does some other part of Pogo prevent them getting here?

Reply:
Yes the characters !#@$ are prevented from the user interface. POGO UI does not allow to enter these characters as part of Enum labels. Whereas for +=* are not prevented from UI so handled them as part of code.